src/main/webapp/app/complaints/complaints-for-students/complaints-student-view.component.ts

```diff
@@ -149,4 +151,20 @@ export class ComplaintsStudentViewComponent implements OnInit {
         }
         return false;
     }
+
+    /**
+     * Opens the complaint form (for a given complaint type) for the student to submit a complaint and scrolls to the bottom of the page to make the form visible
+     */
+    startComplaint(complainType: ComplaintType): void {
+        this.formComplaintType = complainType;
+        // Wait for the view to update
+        setTimeout(() => this.scrollToComplaint(), 0);
+    }
+
+    /**
+     * Scrolls to the complaint form
+     */
+    private scrollToComplaint(): void {
+        this.complaintScrollpoint.nativeElement.scrollIntoView({ behavior: 'smooth' });
```

Comment:
Since the element referred here is rendered conditionally, (see resp. html line 20), a suggestion would be to add the "?" construct to check whether nativeElement (or complaintScrollpoint) really exists to prevent runtime errors.